Services/Mail/service.xml

```diff
@@ -4,6 +4,10 @@
 	<baseclasses>
 		<baseclass name="ilMailGUI" dir="classes" />
 	</baseclasses>
+	<gsproviders>
```

Comment:
I am not sure if we still need this. Could you please check whether or not this is necessary when using the artifact building with `composer dump-autoload`.